packages/lib/src/components/Ach/components/AchInput/types.ts

```diff
@@ -58,4 +58,11 @@ export interface ACHInputProps {
     styles?: StylesObject;
     type?: string;
     forceCompat?: boolean;
+    setComponentRef?: (ref) => void;
+}
+
+// An interface for the members exposed by a component to its parent UIElement
+export interface AchRef {
+    showValidation?: (who) => void;
+    setStatus?: any;
```

Comment:
I believe the setStatus  could be defined as:
```ts
setStatus?(status: UIElementStatus): void
```
I added similar type definition for the UPI pm a while ago, which I expect to be the case for all payment methods

Reply:
ok, cool